tests.py

```diff
@@ -54,9 +42,21 @@ def tearDownClass(self):
     "SFTWindowBeta": 0.001,
     "randSeed": 42,
     "Band": None,
-    **default_signal_params,
 }
 
+default_signal_params = {
+    "F0": 30,
+    "F1": -1e-10,
+    "F2": 0,
+    "h0": 5,
+    "cosi": 0,
+    "psi": 0,
+    "phi": 0,
+    "Alpha": 5e-3,
+    "Delta": 1.2,
+}
+
+
 default_binary_params = {
```

Comment:
Putting this here to get a nice thread view:

> Rodrigo: We could keep the same strategy as for isolated by choosingasini and period parameters such that the binary modulation resembles a very faint spindown; this would make tests pass; but that also implies tp (which, remember, is a phase) becomes degenerated, breaking the second test.

> David: "The second test" = which one?

> Rodrigo: Ups: I mean the test for tp being properly parsed.

I don't quite follow - are you saying some different choice of these parameters (specifically tp) would make the `TestBinaryModulatedWriter::test_tp_parsing` fail? But _not_ because the parser itself fails, but just because that test actually runs a MCMC and that won't recover the value as expected if there isn't actually sufficient binary modulation to resolve it?

Reply:
If `asini` and `period` are such that the signal resembles an isolated signal over the observing time (big `asini`, low `period`), then `tp` becomes degenerated and the MCMC will not recover the injected value.

Reply:
Ok. In that sense the test is overkill, but it seems to pass at the moment, so all good I guess.